src/windows/sid.rs

```diff
@@ -90,9 +90,9 @@ impl Sid {
             if LookupAccountSidW(
                 PCWSTR::null(),
                 sid,
-                PWSTR::from_raw(name.as_mut_ptr()),
+                Some(PWSTR::from_raw(name.as_mut_ptr())),
                 &mut name_len,
-                PWSTR::null(),
+                Some(PWSTR::null()),
```

Comment:
Shouldn't it be `None`?

Reply:
It's strange because you can construct a NULL `PWSTR`. Not sure if they didn't update the API yet or if it's something else...

Reply:
Seems like None works well for all the cases.